pytest_splunk_addon/standard_lib/addon_parser/__init__.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""
+The module provides the Add-on parsing mechanism. It can
+parse the knowledge objects from an Add-on's configuration files
+
+Supports: fields from props & transforms, tags, eventtypes
+
+Dependencies: 
+    splunk_appinspect.App: To parse the configuration files 
+"""
+import os
+import re
+import logging
+from splunk_appinspect import App
+
+from .fields import convert_to_fields, Field
+from .transforms_parser import TransformsParser
+from .props_parser import PropsParser
+from .tags_parser import TagsParser
+from .eventtype_parser import EventTypeParser
+
+LOGGER = logging.getLogger("pytest-splunk-addon")
+
+
+class AddonParser(object):
+    """
+    Parse the knowledge objects from an Add-on's configuration files.
+    Supports: fields from props & transforms, tags, eventtypes
+
+    Args:
+        splunk_app_path (str): Path to the Splunk App
+    """
+    def __init__(self, splunk_app_path):
```

Comment:
missing docstring.

Reply:
I have added the __init__ docstring in the class level docstring as per the Google style docstrings. reference: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html